src/scss/tables.scss

```diff
@@ -72,6 +72,16 @@
         }
     }
 
+    thead {
+        tr > th {
+            cursor: pointer;
```

Comment:
We use `<th>` in other places as well, and we don't want to affect those. I suggest using a class here.